guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,49 @@
+# Faceted Search
+
+Faceted search is a feature provided out-of-the-box by MeiliSearch. Faceting allows to classify search results into categories that are called **facets**.
+
+> For a movie, its title, director, and its release date can be used as facets.
+
+The user can then narrow down search results by selecting facets, which is similar to applying filters to refine the results.
+
+Both faceting and filtering help narrow down the search results. However, **faceting differs from [filtering](/guides/advanced_guides/filtering.md)**.
+
+Faceted attributes **must be declared at indexing time**.
+
+## Using Facets
+
+In order to implement the all new facets, we at meilisearch have added a new attribute `attributesForFaceting`. With the default value of `Null` it accepts `[String]`.
+It should be noted that passing an empty array will recet all faceting attribute in other words any call to settings with a value for the attribute attributesForFaceting overwrites the currently set faceting attributes.
+
+A call to [POST]/indexes/:index_uid/settings should list the currently set facets.
+
+## Using Search Query
+
+When doing a search query, you can specify:
+
+1. The facets to filter on with the facet syntax (explained below), thanks to the optional parameter facetFilters.
+2. the facets for which to retrieve the matching count for, with the facets optional parameter.
```

Comment:
This is unclear, and I think we can blame my specs for that. What we want to say is that `facets` return the number of remaining candidates for each specified facet

```suggestion
2. the facets for which to retrieve the matching count for, with the `facets` optional parameter.
```